Source/ProceduralFairings/FairingBase.cs

```diff
@@ -1123,14 +1130,24 @@ private void FillProfileOutline(PayloadScan scan)
             }
         }
 
+        private void RecalcMaxSize(PayloadScan scan)
+        {
+            if (Mode != BaseMode.Payload)
+            {
+                maxFairingSize = baseSize;
+            }
+            else if (!autoShape)
+            {
+                maxFairingSize = manualMaxSize;
```

Comment:
Hmm, this will always get overwritten with the calculation 2 lines below. Probably not intended?

Reply:
Right, this is a bug. Will fix.